parts/k8s/addons/container-monitoring.yaml

```diff
@@ -22,15 +22,15 @@ metadata:
     addonmanager.kubernetes.io/mode: Reconcile
 ---
 kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Does this change work across all currently supported Kubernetes versions? (1.15-1.19)

Reply:
Yes this works.